app/controllers/api/application_plans_controller.rb

```diff
@@ -27,11 +22,11 @@ def edit
   # to create plan same way as all plans
   #
   def create
-    super params[:application_plan]
+    super params.require(:application_plan)
```

Comment:
This is scary :) I know it was there forever

Ruby wise: the children does not include parent method signature
Rails wise: the action of the parent controller takes arguments.